forem/forem
# API V1 Users docs

## What type of PR is this? (check all applicable)


## Description

Implements the missing users API V1 docs

## Related Tickets & Documents

- Closes #18749

## QA Instructions, Screenshots, Recordings

Execute

```
SWAGGER_DRY_RUN=0 RAILS_ENV=test rails rswag PATTERN="spec/requests/api/v1/**/*_spec.rb"
```

Then check out `swagger/v1/api_v1.json` on https://editor.swagger.io

### UI accessibility concerns?

N/A

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: This PR is part of the API docs milestone that will be communicated once completed

## [optional] What gif best describes this PR or how it makes you feel?

![perfect users](https://media.giphy.com/media/l2JhuVLxujFbV6iRy/giphy.gif)
